whitesource/component.py

```diff
@@ -23,23 +23,24 @@ def get_scan_artifacts_from_components(
     components: typing.Sequence[cm.Component],
 ) -> typing.Generator:
     for component in components:
-        for source in component.sources:
-            if source.type is not cm.SourceType.GIT:
-                raise NotImplementedError
-
-            if source.access.type is not cm.AccessType.GITHUB:
-                raise NotImplementedError
-
-            ws_hint = _get_ws_label_from_source(source)
-
-            if not ws_hint or ws_hint.policy is dso.labels.ScanPolicy.SCAN:
-                yield dso.model.ScanArtifact(
-                    access=source.access,
-                    label=ws_hint,
-                    name=f'{component.name}_{source.identity(component.sources)}'
-                )
-            elif ws_hint.policy is dso.labels.ScanPolicy.SKIP:
+        for source in component.sources + component.resources:
```

Comment:
naming-nit: we call the "base-class" of sources and resources `artifacts` in CNUDIE-CD